src/__tests__/reducers/tasksReducer.test.js

```diff
@@ -0,0 +1,60 @@
+import tasksReducer from '../../redux/reducers/tasksReducer'
+import { SET_SETTINGS, SET_ANSWERS, } from '../../constants'
+
+describe('tasksReducer', () => {
+  test('should return a correct state after receiving a nonexistent action', () => {
+    const action = { type: 'NON_EXISTENT_ACTION', }
+    const initialState = {
+      allTags: true,
+      tagsForTask: [],
+      amountOfWords: 5,
+      random: false,
+      allAnswers: 0,
+      rightAnswers: 0,
+    }
+
+    expect(tasksReducer(undefined, action)).toStrictEqual(initialState)
+  })
+
+  test('should change allTags, random, tagsForTask and amountOfWord properties in the state after receiving the SET_SETTINGS action', () => {
```

Comment:
_should apply user settings_?

Reply:
If you want to test every field, better do it separately

Reply:
maybe